packages/create-ripple-app/template/_example/components/examples/Button.vue

```diff
@@ -0,0 +1,18 @@
+<template>
```

Comment:
Not sure where to finally add these examples in the future, create-app or example site. Can you please put some thoughts here?

Reply:
I've added a script which will copy example content from the reference site, update process would be to create a new site with create-ripple-app, do changes and then run script to update CRA

Reply:
Hi @dylankelly How would you suggest we run this `update.sh`? Manually before each release?